h/static/scripts/controllers/lozenge-controller.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 var Controller = require('../base/controller');
+var SearchTextParser = require('../util/search-text-parser');
```

Comment:
Only constructor function names / class names should be capitalized. If you are importing a module use `camelCase`